resources/assets/sass/_profile.scss

```diff
@@ -7,6 +7,11 @@
 		padding: 60px 40px 40px;
 		margin-bottom: 30px;
 		box-shadow: 2px 2px 4px $gray-300;
+
+		.profile_summary{
+			color: $gray-600;
+			font-family:'Trebuchet MS', 'Lucida Sans Unicode', 'Lucida Grande', 'Lucida Sans', Arial, sans-serif
+		}
```

Comment:
Drop the family declaration, try a darker gray and a lighter weight with slightly larger font-size. Also, even out the margins.

```css

/* _profile.scss */

.profile_summary {
  color: $gray;
  font-weight: 300;
  font-size: 1.05rem;
  margin-top: 1.7rem;
}
```